BLUESPAWN-common/headers/common/wrappers.hpp

```diff
@@ -78,9 +40,24 @@ class HandleWrapper : public GenericWrapper<HANDLE> {
 		GenericWrapper(handle, std::function<void(HANDLE)>(CloseHandle), INVALID_HANDLE_VALUE){};
 };
 
+class FindWrapper : public GenericWrapper<HANDLE> {
+public:
+	FindWrapper(HANDLE handle) :
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Class 'FindWrapper' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=5092780)